src/components/flat-table/flat-table.stories.mdx

```diff
@@ -198,6 +199,115 @@ Any cells preceding the `FlatTableRowHeader`s will automatically also be made st
   </Story>
 </Canvas>
 
+### Using row headers to create left and right hand side sticky columns
+
+In order to have right hand side sticky columns you should set the `FlatTableRowHeader` `stickyAlignment` prop to `right`.
+Any columns to following it will also be sticky.
+
+By default the left hand side columns have the prop set to `left`.
+It is important to make sure that the left hand side columns are not rendered before the right hand side ones.
```

Comment:
I can't tell if there is some ambiguity here, are you referring to the case where columns that should precede the left-aligned sticky column being rendered after the sticky column by mistake, or that for cases with complex tables the render times for the columns should be accounted for so the styles are applied correctly?

Reply:
It means do not render cells that you want to be sticky on the lhs of the table after the cells you want to be sticky on the rhs, I'll try and update this but I was struggling find a succinct way of describing it tbh